### PR TITLE
Update for consistency, added more error checks, updated PPP.

### PR DESCRIPTION
**Pull request by OngDeZhi**
Update: 
1. User Guide for consistency.
2. Developer Guide for consistency.
3. Variable namings to be consistent.
4. PPP.
5. input.txt and EXPECTED.TXT.

Added:
1. More error checks for WatchlistParser and EstimateParser.
2. More error checks for EstimateCommand.
3. More error checks for WatchlistStorage.

Refactor:
1. WatchlistCommand.

### CODE REVIEW

File: docs/team/ongdezhi.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+*   **New Feature:** Added the ability to save and load watchlist data automatically.
+    *   What it does: Helps the user to automatically save their watchlist data into the file `watchlist.txt`, and loads the data found in this file automatically when the application is launched.
+    *   Justification: Being an application to help users keep track of their animes and ensure they stay organized, persistent storage would be needed otherwise, users would have to re-enter their data every time they use the application and that can result in a frustrating experience.
+    *   Highlights: During the loading of the watchlist data, each entry recorded in `watchlist.txt` would be validated to ensure they are not corrupted. **If a watchlist is corrupted, then only that watchlist will not be loaded**, the rest of the watchlist(s) will still be loaded if they are valid. A corrupted watchlist is defined as one that has information missing, has an invalid format, or it contains anime ids that are not found in our database.
+
+*   **Code Contributed:** [RepoSense Link](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=ongdezhi)
```
Comment (n3wsoldier) [inline docs/team/ongdezhi.md]: This link looks nicer but would it be better to provide the link with your code contribution in the code panel.

**Review by n3wsoldier — APPROVED** [review]
LGTM :)

**Review by michaeldinata — APPROVED** [review]
LGTM!!

File: docs/team/ongdezhi.md (hunk 1 above)
Comment (OngDeZhi) [reply docs/team/ongdezhi.md]: Thanks for the suggestion, I will update the link!

**Pull request merged**